js/templates/pageNav.html

```diff
@@ -1,12 +1,51 @@
-<div class="windowControls windowControlsMac">
-  <a class="navClose js-navClose">
-    <i class="ion-ios-close-empty"></i>
-  </a>
-  <a class="navMin js-navMin">
-    <i class="ion-ios-minus-empty"></i>
-  </a>
-  <a class="navMax js-navMax">
-    <i class="ion-ios-plus-empty"></i>
-  </a>
-</div>
-<input type="text" placeholder="Type a handle or OpenBazaar ID" />
\ No newline at end of file
+<header class="borderStacked colBorder">
+  <nav class="bar colPrimary">
+    <div class="flexRow">
+      <div class="windowControlsHolder">
+        <div class="windowControls windowControlsMac">
+          <a class="navClose js-navClose">
+            <i class="ion-ios-close-empty"></i>
+          </a>
+          <a class="navMin js-navMin"></a>
+          <a class="navMax js-navMax"></a>
+        </div>
+      </div>
+      <div>
+        <div class="flexVCent">
+          <a class="iconBtn ion-chevron-left"></a>
+          <a class="iconBtn ion-chevron-right"></a>
+          <a class="iconBtn ion-refresh"></a>
+        </div>
+      </div>
+      <div class="rowDivV colBorderBackground"></div>
+      <div>
+        <div class="flexVCent">
+          <span class="iconBtn">
+            <i class="ion-search"></i>
+          </span>
+        </div>
+      </div>
+      <div class="flexGrow">
+        <div class="flexVCent">
+          <input type="text" class="flexGrow" placeholder="Type a handle or OpenBazaar ID" />
+        </div>
+      </div>
+      <div class="rowDivV colBorderBackground"></div>
+      <div>
+        <div class="flexVCent">
+          <a class="iconBtn ion-android-notifications"></a>
+          <a class="discSm border colBorder"
+          <% if(ob.navimage){ %>
+          //this will be updated to the real image parameter
+          style="background-image: url(ob.navImage), url('imgs/defaultUser.png')"
+          <% } else { %>
+            style="background-image: url('imgs/defaultUser.png')"
```

Comment:
Hmm... those errors are IDE specific. Mine doesn't throw them. Would it be a big problem to disable them?

I don't think copying the image files into .tmp is a bad idea, but I don't think it should be a manual thing. I can integrate it into the tooling, but to do it right we'd need to watch the images folder and then copy over any new / updated images and make sure the sequencing is right with the other tooling (browsersync, etc...). It just takes a bit of work and I think we might have bigger fish to fry (especially if this is to accomodate a specific IDE - for all I know, my IDE might be the minority by not flagging such things...).